dist/angular-actioncable.js

```diff
@@ -107,8 +107,10 @@ ngActionCable.factory('ActionCableController', function (ActionCableConfig) {
   var route = function(message){
     if (!!actions[message.type]) {
       actions[message.type](message);
+      if (message.type == 'ping') methods.after_ping_callback();
     } else if (message.identifier == '_ping') {     // Rails5.0.0.beta3 backport
       actions._ping(message);                       // Rails5.0.0.beta3 backport
+      methods.after_ping_callback();                // Rails5.0.0.beta3 backport
```

Comment:
`after_ping_callback` is not in `actions` because actions can be overwritten; as it is needed to maintain the open connection I put them here...